docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst

```diff
@@ -203,7 +203,19 @@ The Observation Parameter
 =========================
 
 ``o`` is the type used to represent ``Observation``\ s, which are time-dependent quantities that can
-be observed at any given time (such as the “EURUSD” exchange rate in the example above).
+be observed at any given time (such as the “EURUSD” exchange rate in the example above). There are
+two ways of creating an observation in the library:
+
+- :ref:`Observe <constr-contingentclaims-core-observation-observe-30391>` is the original Claims
+  node described in the paper. It does not take a time parameter. Instead, you need to put a
```

Comment:
You don't really need a `When` node in front of it. 

The observable is simply evaluated at the contract's acquisition time. 

The `When` is a way to fix the acquisition time, but in general an `Anytime` node also fixes an acquisition time